### PR TITLE
*: fix 20 broken links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix 17 broken links

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1624

### Which version does your change affect? 

v3.0/v2.1/dev

### CODE REVIEW

File: dev/releases/21rc5.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ category: Releases
 
 ## Tools
 
-- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](https://github.com/pingcap/docs-cn/blob/master/v2.1/reference/tidb-binlog-overview.md)
+- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](https://pingcap.com/docs-cn/dev/reference/tools/tidb-binlog/overview/)
```
Comment (yikeke) [inline dev/releases/21rc5.md]:
```suggestion
- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](/dev/reference/tools/tidb-binlog/overview.md)
```

File: v3.0/releases/21rc5.md [hunk 2]
```diff
@@ -62,4 +62,4 @@ aliases: ['/docs-cn/releases/21rc5/']
 
 ## Tools
 
-- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](https://github.com/pingcap/docs-cn/blob/master/v2.1/reference/tidb-binlog-overview.md)
+- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](https://pingcap.com/docs-cn/v3.0/reference/tools/tidb-binlog/overview/)
```
Comment (yikeke) [inline v3.0/releases/21rc5.md]:
```suggestion
- TiDB 支持 TiDB Binlog cluster，不兼容旧版本 TiDB Binlog [#8093](https://github.com/pingcap/tidb/pull/8093)，[使用文档](/v3.0/reference/tools/tidb-binlog/overview/)
```

**Comment by CaitinChen** [comment]
@yikeke Thanks. PTAL again.

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ category: reference
 
 # TiDB 配置文件描述
 
-TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/config.toml.example](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) 找到默认值的配置文件，重命名为 `config.toml` 即可。本文档只介绍未包含在[命令行参数](/dev/reference/configuration/tidb-server/configuration.md)中的参数。
+TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/config.toml.example](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) 找到默认值的配置文件，重命名为 `config.toml` 即可。本文档只介绍未包含在[命令行参数](https://pingcap.com/docs-cn/dev/reference/configuration/tidb-server/configuration)中的参数。
```
Comment (CaitinChen) [inline dev/reference/configuration/tidb-server/configuration-file.md]: The original relative link was correct, but it caused a 404 error. Therefore, I changed it to an absolute link. @YiniXu9506

**Pull request merged**